content/en/Platform Deep Dive/Credits/_index.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
I was wondering why this looked so familiar, and then I remembered you telling me about how you worked from branches....

To minimize the risk of conflicts, it's a "best practice" to run different PRs from the same branch. But what you've done here is fine, especially since I see you've just rebased.